client/my-sites/customer-home/main.jsx

```diff
@@ -69,6 +70,35 @@ class Home extends Component {
 		isSiteEligible: PropTypes.bool.isRequired,
 	};
 
+	state = {
+		renderChecklistCompleteBanner: null,
+	};
+
+	static getDerivedStateFromProps( nextProps, prevState ) {
+		// If we still don't have checklist data or we don't want to render the banner, probably because
+		// this is a page load, don't change the state.
+		if (
+			null === nextProps.isChecklistComplete ||
+			'norender' === prevState.renderChecklistCompleteBanner
+		) {
+			return null;
```

Comment:
I was getting a warning about the component `state` being undefined. Should we return `prevState` here?

Reply:
Did something changed in React? Last I remember was that state was returning null in getderivedstatefromprops was like saying that the state didn't change.
Will address this.